src/bot.py

```diff
@@ -192,15 +196,50 @@ def is_last_run(self):
                 found_unfinished_run = True
                 break
         return not found_unfinished_run
+    
+    def _rebuild_as_asset_to_trigger(trigger_to_assets: dict):
+        result = {}
+        for key in trigger_to_assets.keys():
+            for asset in trigger_to_assets[key]:
+                result[asset] = key
+        return result
+
+    def on_init(self):
+        keyboard.release(self._config.char["stand_still"])
+        transition_to_screens = Bot._rebuild_as_asset_to_trigger({
+            "select_character": [
+                "MAIN_MENU_TOP_LEFT","MAIN_MENU_TOP_LEFT_DARK"
+            ],
+            "start_from_town": [
+                 "A5_TOWN_0", "A5_TOWN_1",
+                 "A4_TOWN_4", "A4_TOWN_5",
+                 "A3_TOWN_0", "A3_TOWN_1",
+                 "A2_TOWN_0", "A2_TOWN_1", 
+                 "A2_TOWN_10","A1_TOWN_1", "A1_TOWN_3"],
+        })
+        match = self._template_finder.search_and_wait(list(transition_to_screens.keys()), best_match=True)
+        self.trigger_or_stop(transition_to_screens[match.name])
+    
+    def on_select_character(self):
```

Comment:
This will select char now every time a new game is created. Meh : /
We probably want to have some function in char_selector that detects if the correct char is already highlighted...

Reply:
I have this already developed. Checks for highlighted box, crops out unique character name. Then can just template match from then on. I can either make a PR after this is merged or add to this PR once everything's good to go.

Reply:
I think I didn't change this behavior in my PR and kept it as it was. So I suggest adressing this in a separate PR

Reply:
Previously it only selected character after a d2 restart and if a template had been previously saved, so if this PR causes a selection every game creation then that's a change

Reply:
@mgleed Do you want me to add you as collaborator on my fork for this fix? or you can rebase on this in a separate PR? Or you can give me a patch and I'll apply and test? Let me know what works best for you to get your fix on top of this

Reply:
resolved with commit [a2412c0](https://github.com/aeon0/botty/pull/494/commits/a2412c05a6390e414292339c0fc169209a993de0)